tests/python_tests/helpers/golden_generators.py

```diff
@@ -201,6 +202,14 @@ def _gelu(self, x):
         )
         return torch.nn.functional.gelu(input_tensor).item()
 
+    def _fill(self, x):
+        input_tensor = (
+            x
+            if isinstance(x, torch.Tensor)
+            else torch.tensor(x, dtype=format_dict[self.data_format])
+        )
+        return input_tensor.fill_(5).item()
```

Comment:
shouldn't this be x?
```suggestion
        return input_tensor.fill_(x).item()
```

Reply:
x will be the input tensor right? the x will be replaced by scalar (5 in this case)